model/model.wl

```diff
@@ -426,64 +508,64 @@ evaluateScenario[state_, fitParams_, sims_, stateParams_, scenario_]:=Module[{
 		If[KeyExistsQ[events, "cutoff"], events["cutoff"][[1]][[1]],
 	    endOfYear]];
 	
+	sims=generateSimulations[numberOfSimulations,fitParams,standardErrors,endOfEval,stateParams];
+
 	(* generate a solution for each simulation so we can get bands *)
-    simResults = CovidModel[
-        #["r0natural"],
-        #["daysUntilNotInfectiousOrHospitalized"],
-        #["daysFromInfectedToInfectious"],
-        #["daysUntilNotInfectiousOrHospitalized"],
-        #["daysToLeaveHosptialNonCritical"],
-        #["pPCRNH"],
-        #["pPCRH"],
-        #["daysTogoToCriticalCare"],
-        #["daysFromCriticalToRecoveredOrDeceased"],
-        #["fractionOfCriticalDeceased"],
-        #["importtime"],
-        #["importlength"],
-        #["initialInfectionImpulse"],
-        endOfEval,
-        stateParams["params"]["pS"],
-	    stateParams["params"]["pH"],
-	    stateParams["params"]["pC"],
-        containmentThresholdRatio0,
-        stateParams["icuCapacity"],
-        distance,
-        stateParams["hospitalCapacity"]
-    ]&/@sims;
-    
-    deciles = {1/10,2/10,3/10,4/10,5/10,6/10,7/10,8/10,9/10};
+	(* Quiet because we handle bad interpolating functions in the next line *)
+	rawSimResults=Apply[pfunODE2, #]&/@sims//Quiet;
```

Comment:
Ah, I think this answers my question about the array v association from above.

Reply:
Yea...